pkg/accumulator/accumulator_interface.go

```diff
@@ -0,0 +1,8 @@
+package accumulator
+
+type AcccumulatorClient interface {
+	// this function StartCacheAccumulator will store the data from the ebpf engine
```

Comment:
```suggestion
	// StartCacheAccumulator will store the data from the ebpf engine
```
Well, you are obviously referring to the function below, no need to mention it.